README.md

```diff
@@ -1,3 +1,20 @@
-# cascade-layers-polyfill
+# cascade-layers-polyfill-demo
 
-Run `npx webpack` to build the project after clonining. Navigate to the `index.html` of the project in your browser of choice to view.
+This is a demo showing how the Cascade Layers Polyfill is used.
```

Comment:
@mirisuzanne @sanajaved7 @oluoluoxenfree  too vague?

Reply:
maybe just add links to the layers article, and the polyfill? Oh, I see you have those later, which is also fine. But they could work here.